docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -7,4 +7,108 @@
 :keywords: {ml-init}, {stack}, {nlp}, ELSER
 :description: ELSER is a learned sparse ranking model trained by Elastic.
 
-[TBD]
\ No newline at end of file
+experimental[]
+
+Elastic Learned Sparse Encoder - or ELSER - is a text expansion model that uses 
+sparse vector representation and trained by Elastic . As a retrieval model, 
+ELSER performs text-expansion for more relevant search results with expanding a 
+search query to include terms that are not present in the query itself, but 
+semantically highly relevant.
+
+You can use ELSER for various NLP tasks, such as 
+{ref}/semantic-search-elser.html[semantic search]. ELSER is an out-of-domain 
+model which doesn't need to be fine-tuned on your data - it provides relevant 
+search results out of the box.
+
+// TO DO: Refer to this blog post, to learn more about ELSER.
+
+
+[discrete]
+[[elser-req]]
+== Requirements
+
+To use ELSER, you must have the {subscriptions}[appropriate subscription] level 
+or the trial period activated.
+
+
+[discrete]
+[[download-elser]]
+== Download ELSER
+
+You can download ELSER either from the Trained Models page in {kib} or by using 
+the Dev Console.
```

Comment:
Also from Enterprise Search - see https://github.com/elastic/kibana/pull/156080.

I am not sure how heavily we should push people towards the Enterprise Search app in the ML docs, but Enterprise Search is where we're planning to make the semantic search experience as friendly as possible, so we should certainly mention it here.

Reply:
I'm adding a section that covers this method, too.